LachlanMcCaskill/COS30018_Group_I_Option_A_Topic_2_Delivery_Vehicle_Routing
# Save & load functionality

## How to test
Open the `LoadFromFile` scene and the project should load from the file. The file "SceneData.json" is located in the root of the project, you should be able to modify it and re-load the scene to see changes.

Saving is implemented but is currently unused, will probably be used later to generate the "SceneData.json" file if it is missing.

## Improvements that could be made
- Error checking if file is invalid or does not exist
- Automatically generating the file if it does not exist (I might add this before the PR goes through if we think its necessary)
